docs/setting-up/client/mongodb.md

```diff
@@ -180,13 +180,18 @@ pmm-admin add mongodb --tls \
 --tls-certificate-key-file=PATHTOCER \
 --tls-certificate-key-file-password=IFPASSWORDTOCERTISSET \
 --tls-ca-file=PATHTOCACERT
+--authentication-mechanism=AUTHENTICATION-MECHANISM
+--authentication-database=AUTHENTICATION-DATABASE
 ```
 
 where:
 
 - `PATHTOCERT`: Path to TLS certificate file.
 - `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
 - `PATHTOCACERT`: Path to certificate authority file.
+- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use MONGODB-X509 for ssl certificates.
```

Comment:
```suggestion
- `AUTHENTICATION-MECHANISM`: Authentication mechanism. Default is empty. Use `MONGODB-X509` for SSL certificates.
```

Reply:
@percona-csalguero are there any other possible values here? Where or how would a user find out what they should use?

Reply:
The default is `SCRAM` and `MONGODB-X509` is for X509 certs.
Other values depend on the server config: https://docs.mongodb.com/manual/reference/parameters/#mongodb-parameter-param.authenticationMechanisms